evil-collection.el

```diff
@@ -166,6 +166,24 @@ this confusing. It will be included if
   :type '(repeat (choice symbol sexp))
   :group 'evil-collection)
 
+(defcustom evil-collection-key-whitelist '()
+  "List of keys that may be used by evil-collection.
```

Comment:
Spell it "Evil Collection"?

Reply:
You mean refer to the package as Evil Collection in all docstrings?

Reply:
I think so, the README uses "Evil Collection" as a title.